### PR TITLE
gui/settings: add arrow for scroll in theme list.

### PR DESCRIPTION
**Pull request by Zangetsu38**
# About:
is same like for app list
Is very useful for can scroll without wheel, without mouse, iwht just keyoard and click on arrow

# Result:
image

### CODE REVIEW

File: vita3k/gui/src/settings.cpp [hunk 1]
```diff
@@ -590,6 +588,9 @@ static const std::vector<std::string> LIST_SYS_LANG = {
     "Dansk", "Norsk", "Polskis", u8"Português (Brasil)", "English (United Kingdom)", u8"Türkçe"
 };
 
+static const ImU32 ARROW_COLOR = 4294967295; // White
```
Comment (pent0) [inline vita3k/gui/src/settings.cpp:591]: Better to be in hexcimal

File: vita3k/gui/src/settings.cpp [hunk 2]
```diff
@@ -590,6 +588,9 @@ static const std::vector<std::string> LIST_SYS_LANG = {
...
+static float set_scroll_pos, current_scroll_pos, max_scroll_pos;
```
Comment (pent0) [inline vita3k/gui/src/settings.cpp:592]: These variables are better, to be moved in GuiState

Comment (Zangetsu38) [reply vita3k/gui/src/settings.cpp:592]: No, is variable for settings only keep here

File: vita3k/gui/src/settings.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/gui/src/settings.cpp:591]: i already using this in oher place,this function request ImU32 value only

Comment (pent0) [reply vita3k/gui/src/settings.cpp:591]: 0xFFFFFFFF

Comment (Zangetsu38) [reply vita3k/gui/src/settings.cpp:591]: No i said only using this value no hexa

File: vita3k/gui/src/settings.cpp (hunk 2 above)
Comment (pent0) [reply vita3k/gui/src/settings.cpp:592]: current_scroll_pos is definitely used again from previous state

Comment (Zangetsu38) [reply vita3k/gui/src/settings.cpp:592]: No understund is full fine here

File: vita3k/gui/src/settings.cpp (hunk 1 above)
Comment (pent0) [reply vita3k/gui/src/settings.cpp:591]: Why no hexdacimal? This is for people to recognise this RGB code immediately without doubt.

Comment (Zangetsu38) [reply vita3k/gui/src/settings.cpp]: done

**Review by pent0 — APPROVED** [review]

**Pull request merged**